### PR TITLE
KYC-54 Return version info and only build master branch

### PR DESCRIPTION
**Pull request by jchannon**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,4 +7,7 @@ dotnet: 2.1.500
...
\ No newline at end of file
+    secure: vs4xD4L3b0sdwqfIe3+iOMEQ7/l9ci7GKTCmwR7GCPVlO4e8sfDfLK03RxJ8XQhxM9cTEljxaoW+3Ars18/dXQhPjOWaOzhlpnVfWPxhvvvVx0XE+8gHbKv1TEw6+GPHmkvHikNGk22ncWkSa54S3MXRctyldtNLdIzMhPPKP0tMnf2/LgovlSSnOCJw1K6UWqF6XoWYj+uCKwo7IMKeM23PhjxQYD9fIKwQPezcrDwYoyBYUhNqvNw492O2YmehJC9D5hay84Yh4ykv52nV7KX8QhIAUWo343OfXbJmJPZtEcdluIHdG9UvSB3Mc/wOF+z7EE7oPpybK2krmrhTUPGDd6DWYR/rVPotDiB4bJ9/ouUz0yL6w52pSE8Hb2x+pzBUCpXK6MkD1f9ZybkLjjX3vxFDwesYzztShYOT4ODHMA3ytlGGN/LyoHFyO7MSKPDxcdeiKWtkxx1u0WrOnGEAgr6wI7DiX8jucL89l6bZLLhBbzObNPCG+DlvOHUQE2VnMwLzF3EzFYC0Jk/qQpiVSMyxULH/Wyg4fCrqI8tKGnNZnoxQRoBKSRaeShGgofGnOfzeYuQbdHAFgVH1IkedJQ8VNQjftPqd6Bj3N6jus1hZ31Yr6KbmUitlBxMOSZ/NIbAfaz0D7k0JepnYaxQC7AKa0cOlARApClESXjU=
```
[the remark this answers is not in the document]
Comment (jchannon) [reply .travis.yml]: As I suspected it's an encrypted value https://docs.travis-ci.com/user/notifications/#configuring-slack-notifications

File: src/Features/Status/StatusModule.cs [hunk 2]
```diff
@@ -3,14 +3,36 @@
...
+                        {
+                            InstanceId = Environment.GetEnvironmentVariable("WEBSITE_INSTANCE_ID"),
+                            SiteName = Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME"),
+                            Hostname = Environment.GetEnvironmentVariable("WEBSITE_HOSTNAME"),
+                            SlotName = Environment.GetEnvironmentVariable("WEBSITE_SLOT_NAME")
+                        }
+                        : null
+                };
+
+                context.Response.AsJson(version);
```
Comment (JoeStead) [inline src/Features/Status/StatusModule.cs]: `await` the writing of the response

File: src/Features/Status/StatusModule.cs [hunk 3]
```diff
@@ -3,14 +3,36 @@
...
+                var version = new
+                {
+                    Title = typeof(Program).Assembly.Attribute<AssemblyTitleAttribute>(attribute => attribute.Title),
+                    Version = typeof(Program).Assembly.Attribute<AssemblyInformationalVersionAttribute>(attribute => attribute.InformationalVersion),
+                    Azure = Environment.GetEnvironmentVariable("WEBSITE_INSTANCE_ID") != null
+                        ? new
+                        {
+                            InstanceId = Environment.GetEnvironmentVariable("WEBSITE_INSTANCE_ID"),
+                            SiteName = Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME"),
+                            Hostname = Environment.GetEnvironmentVariable("WEBSITE_HOSTNAME"),
```
Comment (JoeStead) [inline src/Features/Status/StatusModule.cs]: What are all of these?

Comment (jchannon) [reply src/Features/Status/StatusModule.cs]: https://github.com/projectkudu/kudu/wiki/Azure-runtime-environment

Comment (jchannon) [reply src/Features/Status/StatusModule.cs]: It's what ironclad uses

File: build.sh [hunk 4]
```diff
@@ -9,6 +9,7 @@ if dotnet test ./tests/Apollo.Tests.Unit/
         if [[ "$TRAVIS_BRANCH" = "master" ]]; then
           echo "Publishing apollo"
           docker push lykkecorp.azurecr.io/apollo:latest
+          #TODO git tag
```
[the remark this answers is not in the document]
Comment (jchannon) [reply build.sh]: Added a link

[the remark this answers is not in the document]
Comment (JoeStead) [reply build.sh]: 👍

**Pull request merged**